cpp/CMakeLists.txt

```diff
@@ -171,7 +172,7 @@ include(cmake/modules/ConfigureCUDA.cmake)
 ##############################################################################
 # - Set options based on user defined one  -----------------------------------
 set(CUML_USE_RAFT_NN OFF)
```

Comment:
Should this option go away too now?

Reply:
Not quite yet- this is enabling FAISS in the meantime since cuml will depend on it just a tad longer. We'll remove this when we remove FAISS